assessments/intake-process.md

```diff
@@ -0,0 +1,45 @@
+# Security Assessment Priorities & Pipeline Intake Process
+
+SIG-Security has a volunteer team of subject matter experts and industry
+professionals dedicated to helping SIG-Security members, the TOC, and the larger
+CNCF community maintain an understanding of the current state of security in the
+cloud native ecosystem and helping cloud native projects succeed.
+
+The following process describes how projects are prioritized for security
+assessments.
+
+# Authority
+
+* The [Security Assessment
+  Facilitator](../governance/roles.md#security-assessment-facilitator) maintains
```

Comment:
I think this should be a direct link instead of a relative one?

Reply:
not sure what you mean here

Reply:
I was thinking of doing `/governance/roles.md#security-assessment-facilitator` instead of `../`

Reply:
makes sense, will search for `..`

Reply:
done